src/solvers/smt2_incremental/smt2_incremental_decision_procedure.h

```diff
@@ -70,13 +70,20 @@ class smt2_incremental_decision_proceduret final
   /// Sends the solver the definitions of the object sizes.
   void define_object_sizes();
 
+  /// Namespace for looking up the expressions which symbol_exprts relate to.
+  /// This includes the symbols defined outside of the decision procedure but
+  /// does not any SMT function identifiers the decision procedure introduces.
```

Comment:
Missing a verb here?

Reply:
I have updated this. Does it now read better?

Reply:
Yes :+1: